### PR TITLE
DHFPROD-8734: Metadata widget

### PR DESCRIPTION
**Pull request by iveyMarklogic**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
  

- ##### Reviewer:

- [ ] Reviewed Tests
- [ ] Added to Release Wiki

### CODE REVIEW

**Review by wooldridge — CHANGES_REQUESTED** [review]
I also had some edits here: https://github.com/iveyMarklogic/marklogic-data-hub/pull/5

File: marklogic-data-hub-central/ui-custom/src/components/DataTableMultiValue/DataTableMultiValue.tsx [hunk 1]
```diff
@@ -127,6 +127,14 @@ const DataTableMultiValue: React.FC<Props> = (props) => {
                 {data.map((d, i) => {
                 return (
                     <tr key={"row-" + i} className={data.length === 1 ? "singular" : ""}>
+                        {_.isArray(props.config.metadata) && props.config.metadata.map((meta, i2) => {
```
[the remark this answers is not in the document]
Comment (iveyMarklogic) [reply marklogic-data-hub-central/ui-custom/src/components/DataTableMultiValue/DataTableMultiValue.tsx:137]: @wooldridge I fixed that. Can you review it again?

**Review by wooldridge — CHANGES_REQUESTED** [review]
Looks goooooood! One minor fix:
Screen Shot 2022-04-13 at 11 13 23 AM
I narrowed the metadata column with this:
Screen Shot 2022-04-13 at 11 21 02 AM

**Comment by iveyMarklogic** [comment]
@wooldridge I fixed the column issue

**Review by wooldridge — APPROVED** [review]
Looks good!
Screen Shot 2022-04-13 at 1 23 00 PM

**Pull request merged**